_search-plugins/sql/functions.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
 title: Functions
```

Comment:
i see there are sql/sql/functions, sql/ppl/functions (those are actually commands), sql/functions. 

> SQL language supports all SQL plugin functions

> PPL supports all SQL functions

sql/functions are referred as "SQL functions" or "SQL plugin functions", but then there are also SQL specific functions (sql/sql/functions). seems a bit confusing, should `sql/functions` be referred as maybe SQL plugin common (or shared) functions?